.github/workflows/build.yml

```
@@ -0,0 +1,21 @@
+name: Java CI with Gradle
+
+on:
+  push:
+    branches: [ $default-branch ]
+  pull_request:
+    branches: [ $default-branch ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    matrix:
+      java: [6, 8, 11, 15]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ matrix.java }}
+      - name: gradlew check
```

I'd prefer a more natural sounding name, like so:
```suggestion
      - name: Run checks
```